### PR TITLE
Use default vagrant-libvirt provider options for libvirt conn uri and switch to official openSUSE Tumblweed boxes

### PR DESCRIPTION
**Pull request by toabctl**
See commit messages

### CODE REVIEW

**Comment by toabctl** [comment]
@denisok could you have a look please?

File: README.md [hunk 1]
```diff
@@ -25,6 +25,11 @@ Or use script in case you have openSUSE:
 # sudo ./openSUSE_vagrant_setup.sh
 ```
 
+#### libvirtd on a remote machine
```
Comment (denisok) [inline README.md]: ok, how it works without ssh? Anything needs to be setup before hand?
I go this error if I comment out ssh:
`Error while connecting to libvirt: Error making a connection to libvirt URI qemu:///system?no_verify=1&keyfile=/home/dkondratenko/.ssh/id_rsa:
Call to virConnectOpen failed: authentication unavailable: no polkit agent available to authenticate action 'org.libvirt.unix.manage'`

File: Vagrantfile [hunk 2]
```diff
@@ -18,16 +18,14 @@ config = YAML.load_file(config_file)
 Vagrant::Hosts::check_for_ssh_keys
 
 ### take BOX from the enviroment ###
-BOX = ENV.has_key?('BOX') ? ENV['BOX'] : 'opensuse/openSUSE-42.3-x86_64'
+BOX = ENV.has_key?('BOX') ? ENV['BOX'] : 'opensuse/openSUSE-15.1-x86_64'
```
Comment (denisok) [inline Vagrantfile]: there is no `opensuse/openSUSE-15.1-x86_64` in config.yml

File: README.md (hunk 1 above)
Comment (toabctl) [reply README.md]: Usually your desktop env (eg. gnome-shell) does provide a polkit-agent (which then asks for a password). Which desktop do you use?
To not need to tell the password every time, add yourself to the `libvirt` group.

File: Vagrantfile (hunk 2 above)
Comment (toabctl) [reply Vagrantfile]: fix (not sure why this config has this strange `sub0/XXX` name)

File: README.md (hunk 1 above)
Comment (toabctl) [reply README.md]: @denisok do you want me to add a section to the README that you need a polkit-agent or add yourself to the libvirt group?

Comment (denisok) [reply README.md]: I don't use desktop environment. BTW our CI doesn't either.

File: Vagrantfile (hunk 2 above)
Comment (denisok) [reply Vagrantfile]: that shouldn't be sub0, @colder-is-better would fix it soon.

File: README.md (hunk 1 above)
Comment (denisok) [reply README.md]: the question, why we need this change, if it automated with ssh now ?
imagine 10-20 nodes deployment - why to enter password all the time ?

Comment (toabctl) [reply README.md]: No. The solution is to add yourself to the `libvirt` group. This is simpler than installing, starting and enabling sshd and then ssh-copy-id to localhost. And imo it's nicer to not need a local sshd running.
Is the CI setup somewhere automated? Than I can have a look to add the jenkins user (or whatever user is used) to the libvirt group

Comment (toabctl) [reply README.md]: @denisok I can also drop that commit if you think it's not useful. Just let me know.

**Comment by toabctl** [comment]
I don't plan to work on that.

**Pull request closed without merging**